tools/cold-store/src/cli.rs

```diff
@@ -62,6 +83,16 @@ struct CopyNextBlocksCmd {
     number_of_blocks: usize,
 }
 
+#[derive(Parser)]
+struct CopyAllBlocksCmd {
+    #[clap(short = 'M', long, default_value_t = 0)]
+    max_mem_size: usize,
+    #[clap(short = 'm', long, default_value_t = 0)]
+    min_batch_size: usize,
+    #[clap(short, long)]
+    check_after: bool,
+}
```

Comment:
mM :D 
Can you add nice comments for each? You can just copy paste the comments from above, they are nice.

Reply:
Can you also default check_after to true? What would be sensible defaults for -m and -M?

Reply:
Made checking a default, the flag now means no check, made default batch size ~500MB